tests/test_camera.py

```diff
@@ -12,6 +12,7 @@ def test_init(mocker, m5stickv):
     from krux.camera import Camera
 
     c = Camera()
+    c.initialize_sensor()
```

Comment:
Not necessary to add this call since it's covered by the `test_initialize_sensor` test. This test is just asserting that you can instantiate a Camera without an exception being raised

Reply:
I've changed camera's code later so Camera is not initialized at boot, because there's no need to spend time and power on this at boot since it will be re-initialized later. But that can be reversed too.